tests/resources/testcases/test_cmp/scenario.yaml

```diff
@@ -0,0 +1,10 @@
+samples:
+  sample_a:
+    resolution: 100
+    universe: "[0.0,1.0]"
+  sample_b:
+    resolution: 100
+    universe: "[0.0,1.0]"
+
+events:
+  homhet: "sample_a > sample_b"
```

Comment:
I think your issue comes from the fact that you are not defining an "otherwise" event here. Remember, the event space has to be complete. Otherwise, the model has no other chance than accepting the frequency combinations that you allow here.